scripts/02_reporting/tables.R

```diff
@@ -346,46 +473,168 @@ tab_cost_est_phase1 <- dplyr::left_join(
+                habitat_value,
+                barrier_result,
+                source) |>
+  # we must transform the data to latitude/longitude (CRS 4326)
+  sf::st_transform(4326)
+
+
+
+
+tab_map_phase_1 <- tab_map_phase_1_prep |>
+  dplyr::mutate(priority_phase1 = dplyr::case_when(priority_phase1 == 'mod' ~ 'moderate',
+                                     TRUE ~ priority_phase1),
+                priority_phase1 = stringr::str_to_title(priority_phase1)) |>
+  dplyr::mutate(data_link = paste0('<a href =', 'sum/cv/', pscis_crossing_id, '.html ', 'target="_blank">Culvert Data</a>')) |>
+  dplyr::mutate(photo_link = dplyr::case_when(is.na(my_crossing_reference) ~ paste0('<a href =', 'https://raw.githubusercontent.com/NewGraphEnvironment/', repo_name, '/main/data/photos/', pscis_crossing_id, '/crossing_all.JPG ',
+                                                                                    'target="_blank">Culvert Photos</a>'),
+                                              TRUE ~ paste0('<a href =', 'https://raw.githubusercontent.com/NewGraphEnvironment/', repo_name, '/main/data/photos/', my_crossing_reference, '/crossing_all.JPG ',
+                                                            'target="_blank">Culvert Photos</a>'))) |>
+  dplyr::mutate(model_link = paste0('<a href =', 'sum/bcfp/', pscis_crossing_id, '.html ', 'target="_blank">Model Data</a>')) |>
+  dplyr::distinct(site_id, .keep_all = TRUE) #just for now
+
+
+
+## Phase 2 --------------------------------------------------------------
+
+#please note that the photos are only in those files because they are referenced in other parts of the document
+tab_map_phase_2 <- dplyr::left_join(
+  tab_cost_est_prep9,
+  form_fiss_site |>
+    dplyr::filter(is.na(ef) & location == "us") |>
+    dplyr::select(site, utm_zone, easting = utm_easting, northing = utm_northing, comments),
+  by = c('pscis_crossing_id' = 'site')
+) |>
+  # we must transform the data to latitude/longitude (CRS 4326)
+  fpr::fpr_sp_assign_sf_from_utm() |>
+  sf::st_transform(4326) |>
+  # We don't have a priority ranking for the hab con sites at the moment so just just add the priority from the phase 1 assessments.
+  dplyr::left_join(tab_map_phase_1 |>
+                     sf::st_drop_geometry() |>
+                     dplyr::select(pscis_crossing_id, priority = priority_phase1),
+                   by = 'pscis_crossing_id') |>
 
+  # Update the data link to point to the new location in docs
+  dplyr::mutate(
+    data_link = paste0(
+      '<a href =',
+      'sum/cv/', pscis_crossing_id,
+      '.html ', 'target="_blank">Culvert Data</a>'
+    )
+  ) |>
+  dplyr::mutate(
+    model_link = paste0(
+      '<a href =',
+      'sum/bcfp/', pscis_crossing_id,
+      '.html ', 'target="_blank">Model Data</a>'
+    )
+  ) |>
+  dplyr::mutate(
+    photo_link = paste0(
+      '<a href =',
+      'https://raw.githubusercontent.com/NewGraphEnvironment/fish_passage_skeena_2024_reporting/main/data/photos/',
```

Comment:
https://www.newgraphenvironment.com/ngr/reference/ngr_str_link_url.html

ha!! so much error prone text we may save.  think it should work well